app/bamboo.yml

```diff
@@ -0,0 +1,89 @@
+---
+settings:
+  artifacts-dir: results/bamboo/%Y-%m-%d_%H-%M-%S
+  aggregator: consolidator
+  verbose: false
+  env:
+    application_hostname: test-bamboo.atlassian.com    # Bamboo DC hostname without protocol and port e.g. test-bamboo.atlassian.com or localhost
+    application_protocol: http      # http or https
+    application_port: 8085            # 80, 443, 8080, 4990, etc
+    secure: True                    # Set False to allow insecure connections, e.g. when using self-signed SSL certificate
+    application_postfix:            # e.g. /bamboo in case of url like http://localhost:4990/bamboo
```

Comment:
let's remove 4990 from examples as this is a crowd port

Reply:
Will remote port at all till the moment we will re-check the port from local installation.